.github/workflows/pull-request.yml

```
@@ -296,6 +273,133 @@
+          echo "Waiting for server to stabilize..."
+          sleep 10
+          if curl -f -k https://localhost:4001 | jq -e '. == {"talawa-version":"v1","status":"healthy"}' > /dev/null; then
+            echo "Production server is up and healthy."
+          else
+            echo "Production server is down. Cleaning up..."
+            kill -9 $PROD_PID || echo "Process already stopped."
+            exit 1
+          fi
+
+      - name: Check Apollo Server for Production
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:10:9: Double quote to prevent globbing and word splitting

(shellcheck)